arch/x86/x86_features.c

```diff
@@ -10,7 +10,7 @@
 #include "../../zbuild.h"
 #include "x86_features.h"
 
-#ifdef _WIN32
+#ifdef _MSC_VER
```

Comment:
`_WIN32` should be defined when compiled as 32-bit or 64-bit. Perhaps the toolchain is not properly adding that define? Maybe the toolchain files are better place to add `add_definitions(_WIN32)`.

Reply:
`_WIN32` is defined when using MinGW, however `intrin.h` is a MSVC header - MinGW-w64 provides an implementation of it for compatibility with MSVC but the original MinGW doesn't. This change means that the GCC headers will be used instead.